.github/workflows/dev.workflow.yml

```diff
@@ -0,0 +1,46 @@
+# Build AutoCorrode for PRs
+name: Build AutoCorrode for PRs
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: makarius/isabelle:Isabelle2025
+      options: --user root
+
+    steps:
+      - name: Checkout AutoCorrode
+        uses: actions/checkout@v3
+      
+      - name: Checkout AFP
+        uses: actions/checkout@v3
+        with:
+          repository: isabelle-prover/mirror-afp-2025
+          path: afp
+
+      - name: Set isabelle home 
```

Comment:
Can you hoist the setup steps into a separate action, so we can easily reuse it across jobs?

Reply:
On second thought, let's do this in the follow-up PR solving #4